jersey-tck/pom.xml

```diff
@@ -67,10 +145,140 @@
                         </goals>
                         <configuration>
                             <dependenciesToScan>jakarta.ws.rs:jakarta.ws.rs-tck</dependenciesToScan>
+                            <includes>
+                                 <include>**/tck/**/*IT</include>                            
+                            </includes>
+                            <systemPropertyVariables>
+                            <GLASSFISH_HOME>${project.build.directory}/glassfish6</GLASSFISH_HOME>
+                            <servlet_adaptor>org.glassfish.jersey.servlet.ServletContainer</servlet_adaptor>
+                            <webServerHost>localhost</webServerHost>
+                            <webServerPort>8080</webServerPort>
```

Comment:
Is it essential to have this static or can we go with 0 for auto-scan?

Reply:
For now the webServerHost & webServerPort cannot be null and must be set while running the TCK. I have defaulted the servlet_adaptor to org.glassfish.jersey.servlet.ServletContainer in jaxrs-tck/src/main/java/jakarta/ws/rs/tck/ee/rs/core/request/JAXRSClientIT.java.

Do you have